apps/showcase/src/assets/trainings/sdk/steps/model-extension/instructions.md

```diff
@@ -0,0 +1,41 @@
+There are certain cases in which you want to be able to extend your SDK models, and therefore ensure that revivers are generated.
+
+Let's continue with the use case of the previous exercise.\
+In order to keep track of the user's current booking, it would be useful to generate an ID.
+To do this, we can create a new model which extends the previously generated `Flight` type.
+
+> [!NOTE]
+> For this exercise, we provide an SDK that has been generated with model extension.
+> 
+> To verify this, check out the configuration used to generate the API (either the command line or the file `openapitools.json`).\
+> Ensure that the global property `allowModelExtension` has been set to `true`.\
+> This will guarantee that the revivers of the base models are generated, which is essential for the following exercise.
+
+### Exercise
+
+Extensions of base models (located in the `base` folder) are created in the `core` folder.
+
+#### Creating the extended model
+First, let's create the type `FlightCoreIfy` in `libs/sdk/src/models/core/flight.ts`.
+This type should extend the type `Flight`, imported from the `base` folder and add a new field `id: string`.
```

Comment:
just a suggestion so the user writes the field themselves
```suggestion
This type should extend the type `Flight`, imported from the `base` folder and add a new field `id` of type `string`.
```

Reply:
should we add an example or a template here in the instructions?

Reply:
I want to avoid the exercise being just some copy-paste and replace.
Unless you think this step may be too difficult for some people, I think it's best to let them figure it out.